.github/workflows/update_version_config.yaml

```diff
@@ -0,0 +1,45 @@
+name:  🤖 Automated Update tagpr config
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to release'
+        required: true
+        default: '0.3.0'
```

Comment:
Is it difficult to choose `major`, `minor`, or `patch` to take place of a version?